fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/MfdFmsPerf.tsx

```diff
@@ -963,6 +977,10 @@ export class MfdFmsPerf extends FmsPage<MfdFmsPerfProps> {
           }
         }),
     );
+
+    NXDataStore.getAndSubscribe('CONFIG_USING_METRIC_UNIT', (key, value) => {
+      value === '1' ? this.unit.set(UnitType.METER) : this.unit.set(UnitType.FOOT);
+    });
```

Comment:
```suggestion
    NXDataStore.getAndSubscribe('CONFIG_USING_METRIC_UNIT', (key, value) => {
      this.lengthUnit.set(value === '0' ? UnitType.FOOT : UnitType.METER);
    }, '1');
```

Reply:
I did this initially, and the side-effect is that if you have lbs set on the flypad before navigating to the page, then it will always be in Meters. As the subscribe comes "onAfterRender".